editors/vscode/src/extension.ts

```diff
@@ -41,8 +41,10 @@ function lspPlatform(): Platform | null {
         if (typeof vscode.env.remoteName !== "undefined") {
             remote_env_options = {
                 DISPLAY: ":0",
-                SLINT_FULLSCREEN: "1",
             };
+            if (vscode.env.remoteName != "wsl") {
+                Object.defineProperty(remote_env_options, "SLINT_FULLSCREEN", { value: "1", enumerable: true })
```

Comment:
Is that really what we want to do here?

I think this should be an option in vscode instead of trying to guess this.